rust-support/chatdbg/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "chatdbg"
-version = "##VERSION##"                                 # Sync with pyproject.toml on release.
+version = "0.6.5"                                 # Sync with pyproject.toml on release.
```

Comment:
These `##VERSION##` should stay as-is, they'll get auto-replaced 👍  The only one that needs changing is the main `pyproject.toml` one.

Reply:
I realized the comment is now misleading -- will fix on master.